odoo/addons/base/models/assetsbundle.py

```diff
@@ -722,6 +835,21 @@ def to_node(self):
                 ['data-asset-version', self.bundle.version],
             ]), self.with_header())
 
+    def with_header(self, content=None, minimal=True):
+        if minimal:
+            return super().with_header(content)
+
+        header = [
+            f"*  Filepath: {self.url}",
+            f"*  Bundle: {self.bundle.name}",
+            f"*  Lines: {len(content.splitlines())}",
+        ]
+        length = max(map(len, header)) + 2
+        header = [f'{line:<{length}}*' for line in header]
+        begin = "/" + "*" * (length)
+        end = "*" * (length) + "/"
+        return "\n".join(["", begin, *header, end, content])
```

Comment:
It took me several minutes to figure out what this code was doing.
At first sight I even thought it was buggy.
I suggest to put all the "box" formatting in one place, to make it clear that we add 3 characters on each side:
```suggestion
        # format the header like
        #   /**************************
        #   *  Filepath: <asset_url>  *
        #   *  Bundle: <name>         *
        #   *  Lines: 42              *
        #   **************************/
        lines = [
            f"Filepath: {self.url}",
            f"Bundle: {self.bundle.name}",
            f"Lines: {len(content.splitlines())}",
        ]
        length = max(map(len, lines))
        return "\n".join([
            "",
            "/" + "*" * (length + 5),
            *(f"*  {line:<{length}}  *" for line in lines),
            "*" * (length + 5) + "/",
            content,
        ])
```